generators/server/templates/src/main/java/package/web/rest/ClientForwardController.java.ejs

```diff
@@ -0,0 +1,30 @@
+<%#
+Copyright 2013-2019 the original author or authors from the JHipster project.
+
+This file is part of the JHipster project, see https://www.jhipster.tech/
+for more information.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-%>
+package <%=packageName%>.web.rest;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class ClientForwardController {
+    @GetMapping(value = "/**/{path:[^\\.]*}")
```

Comment:
> all paths that do not contain a period (and are not explicitly mapped already) are Angular routes, and should forward to the home page
https://spring.io/blog/2015/05/13/modularizing-the-client-angular-js-and-spring-security-part-vii